lib/widgets/layouts/enum/adaptive_destinations_enum.dart

```diff
@@ -1,6 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
 enum AdaptiveDestinationEnum {
   contacts,
   rooms,
-  stories,
+  settings,
   search;
+
+  String getTitle(BuildContext context) {
+    switch (this) {
+      case AdaptiveDestinationEnum.contacts:
+        return L10n.of(context)!.contacts;
+      case AdaptiveDestinationEnum.rooms:
+        return L10n.of(context)!.chat;
+      case AdaptiveDestinationEnum.settings:
+        return L10n.of(context)!.settings;
+      case AdaptiveDestinationEnum.search:
+        return '';
+      default:
+        return '';
+    }
+  }
+
+  IconData? getIcon() {
+    switch (this) {
+      case AdaptiveDestinationEnum.contacts:
+        return Icons.contacts_outlined;
+      case AdaptiveDestinationEnum.rooms:
+        return null;
+      case AdaptiveDestinationEnum.settings:
+        return Icons.settings;
+      case AdaptiveDestinationEnum.search:
+        return null;
+      default:
+        return null;
+    }
+  }
+
+  int getActiveBottomNavigationBarIndex() {
```

Comment:
it is not good, if I change order of destination array, this switch case won't work
just use destination.indexOf(this)